src/context/CardAlias/hooks/useCardAliasContext.ts

```diff
@@ -0,0 +1,15 @@
+import { useContext } from 'react';
+
+import { ERROR_MESSAGE } from '@/constants/errorMessage';
+
+import { CardAliasContext } from '../CardAliasProvider';
+
+export default function useCardAliasProvider() {
+  const cardAliasProvider = useContext(CardAliasContext);
+
+  if (cardAliasProvider === undefined) {
+    throw new Error(ERROR_MESSAGE.PROVIDER.INVALID_SCOPE);
+  }
```

Comment:
에러를 검증하는 구문도 if문을 사용하지 않고 별도 함수로 추상화해보면 어떨까요?

```suggestion
  assert(cardAliasProvider != null, ERROR_MESSAGE.PROVIDER.INVALID_SCOPE);
```

Reply:
`assert`로 추상화를 해두고 어플리케이션 전역에서 일관되게 사용하는게 좋은 것 같습니다!